cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java

```diff
@@ -46,7 +50,17 @@ public void receive(String message) {
 		
 		System.out.println("Gradebook has received: "+message);
 
-		//TODO  deserialize message to EnrollmentDTO and update database
+		EnrollmentDTO enrollmentDTO = fromJsonString(message, EnrollmentDTO.class);
```

Comment:
Looks correct, enrollment DTO is received as expected, and course changes the right valued after ensuring it isnt empty